geppetto-client/js/common/layout/WidgetFactory.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+
+import { Widget } from './model';
```

Comment:
not sure about this, I copied all available for the layout manager when we discussed about that, not sure if that code portion moved forward but in case I think we should address that in a separate card.

Reply:
This feature was not implemented in my version of the WidgetFactory because it was directly implemented in the application.  Once it becomes part of the library we need a way to extend the functionality of the WidgetFactory to allow the application to add custom components to the layout.